src/tests/fs_tests.rs

```diff
@@ -582,4 +584,29 @@ mod fs_tests {
         assert_eq!(cage.exit_syscall(), 0);
         lindrustfinalize();
     }
+
+    pub fn ut_lind_fs_rmdir() {
+        lindrustinit();
+        let cage = {CAGE_TABLE.read().unwrap().get(&1).unwrap().clone()};
+
+        let path = String::from("/parent_dir/dir");
+        assert_eq!(cage.mkdir_syscall(&String::from("/parent_dir"), S_IRWXA), 0);
+        assert_eq!(cage.mkdir_syscall(&path, S_IRWXA), 0);
+        assert_eq!(cage.rmdir_syscall(&path), 0);
+
+        assert_eq!(cage.exit_syscall(), 0);
+        lindrustfinalize();
+    }
+
+    pub fn ut_lind_fs_rename() {
+        lindrustinit();
+        let cage = {CAGE_TABLE.read().unwrap().get(&1).unwrap().clone()};
+
+        let old_path = String::from("/test_dir");
+        assert_eq!(cage.mkdir_syscall(&old_path, S_IRWXA), 0);
+        assert_eq!(cage.rename_syscall(&old_path, &String::from("/test_dir_renamed")), 0);
```

Comment:
Remove the &String::from calls, just use a &str literal, and do not borrow old_path